packages/suite/src/components/suite/hocNotification/components/withTransaction.tsx

```diff
@@ -64,20 +69,19 @@ const withTransaction = (View: React.ComponentType<ViewProps>, props: StrictView
             };
         }
 
+        const openTxDetailsModal = (transaction: WalletAccountTransaction, rbfForm?: boolean) => {
+            openModal({
+                type: 'transaction-detail',
+                tx: transaction,
+                rbfForm,
+            });
+        };
+
         return (
             <View
                 {...props}
-                action={async () => {
-                    // select device
-                    await dispatch(suiteActions.selectDevice(device || notification.device));
```

Comment:
I am not sure what was the purpose of this line of code. Maybe I shouldn't have deleted it

Reply:
I think it was for the case where notification was related to other wallet than the one that is currently selected. Eg. you have 2 wallets added - standard and passphrase. You are currently inside the passphrase wallet, but you just received a notification about transaction coming to your standard wallet. In order to redirect you (line 74) to the correct account belonging to the standard wallet we need to switch the selected device to one representing the standard wallet (1 instance of a device = 1 wallet). If I am right then that brings a question if it isn't weird that now you can examine details of tx that may be belonging to different wallet than the one that you have active. Though it is probably fine :)

Reply:
hmmm but then the user is missing the most important information, what is the account that received the transaction 🤷‍♂️

Reply:
It will be further discussed with @matejzak how to properly address this issue.